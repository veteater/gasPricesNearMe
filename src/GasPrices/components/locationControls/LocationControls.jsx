import React from 'react';

const LocationControls = () => {
  return (
    <article className="locationControls">
      <span>Find Near Me</span>
      <span>- or -</span>
      <span>
        <input type="text" value="zipcode" placeholder="Enter a zipcode..." readOnly />
        <span>Find</span>
      </span>
    </article>
  );
};

export default LocationControls;