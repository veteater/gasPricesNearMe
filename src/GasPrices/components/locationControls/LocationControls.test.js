import React from 'react';
import { shallow } from 'enzyme';
import LocationControls from './LocationControls';

describe('LocationControls', () => {
  it('renders without crashing', () => {
    const component = shallow(<LocationControls />);
    expect(component).toMatchSnapshot();
  });
});
