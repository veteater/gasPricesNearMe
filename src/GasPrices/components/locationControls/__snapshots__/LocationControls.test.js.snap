// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationControls renders without crashing 1`] = `
<article
  className="locationControls"
>
  <span>
    Find Near Me
  </span>
  <span>
    - or -
  </span>
  <span>
    <input
      placeholder="Enter a zipcode..."
      readOnly={true}
      type="text"
      value="zipcode"
    />
    <span>
      Find
    </span>
  </span>
</article>
`;
